static/panes/compiler.js

```diff
@@ -1478,9 +1478,6 @@ function getAsmInfo(opcode) {
         return Promise.resolve(cached.found ? cached.result : null);
     }
     var base = window.httpRoot;
-    if (!base.endsWith('/')) {
```

Comment:
I hope this is ok :grin: -- I'm trusting this was just an over-defensive thing from before...

Reply:
Thanks!